src/main/java/com/clevertap/stormdb/StormDB.java

```diff
@@ -571,6 +594,63 @@ public void flush() throws IOException {
         }
     }
 
+    private void readFromDeletedKeysFile(RandomAccessFile file, Consumer<Integer> entryConsumer) throws IOException{
+        ByteBuffer bufferDeletedKeys = ByteBuffer.allocate(4); // will hold 10 keys
```

Comment:
Could you clarify your comment here? It appears that you're allocating just 4 bytes, but the comment refers to 10 keys.

Reply:
I forgot to remove this.